mw-webapp/src/logic/wayPage/reportsTable/WayColumns.tsx

```diff
@@ -132,6 +108,13 @@ const updateDayReportState = (
   setReports(updatedDayReports);
 };
 
+/**
+ * Case when need render confirm
+ */
+export const isRenderConfirm = (confirmUuid: string | undefined, uuid: string) => {
```

Comment:
not clear what the difference betwee uuid an dconfirm uuid and why do we need them both

Reply:
Deleted deprecated code